src/app/drive/services/download.service/downloadFile.ts

```diff
@@ -125,7 +105,16 @@ export default async function downloadFile(
     support = DownloadSupport.PatchedStreamApi;
   }
 
-  trackFileDownloadStart(userEmail, fileId, itemData.name, itemData.size, itemData.type, itemData.folderId);
+  const trackingDownloadProperties: TrackingPlan.DownloadProperties = {
+    file_download_id: analyticsService.getTrackingActionId(),
+    file_id: parseInt(fileId),
```

Comment:
Is this the fileId from the network (string) or the autoincremental id (unsigned int)? Unless the interface is explicitly changed, I will recommend double-checking if this is a string or an int, parseInt could fail and all the `downloadFile` function will crash

Reply:
This is the network fileId. Changes added